Source/Core/UICommon/NetPlayIndex.cpp

```diff
@@ -227,6 +227,7 @@ void NetPlayIndex::Remove()
     return;
 
   m_running.Set(false);
+  m_session_thread_cv.notify_all();
```

Comment:
Does this actually block until the thread is finished / ``wait_for`` is called?

Reply:
No, we still use `join` for that.